.cirrus.yml

```diff
@@ -0,0 +1,10 @@
+task:
```

Comment:
not sure what part of this config runs test ? 
does it have to be `./gradlew testDebugUnitTest` somewhere or it will be triggered automatically, and this is pnly results publishing?

Reply:
`./gradlew check` runs both lint and unit tests 👍